lib/nodes/proxyShapeBase.cpp

```diff
@@ -620,6 +621,8 @@ MayaUsdProxyShapeBase::GetOutputTime(MDataBlock dataBlock) const
 MBoundingBox
 MayaUsdProxyShapeBase::boundingBox() const
 {
+    TRACE_FUNCTION();
```

Comment:
How can this trace log be used?

Reply:
Yeah, it's not that easily discoverable, but the docs for the trace library cover this a little bit under the "Recording and Reporting" section on this page:
https://graphics.pixar.com/usd/docs/api/trace_page_front.html

Here's a little incantation for the script editor that captures trace library calls:
```
from pxr import Trace
collector = Trace.Collector()
collector.enabled = True

# Do some stuff here, like a viewport draw, for example.
cmds.refresh()

# Dump the results to a file.
Trace.Reporter.globalReporter.Report('trace.txt')
```

Reply:
Very useful, thank you!